tests/test_telemetry_monitor.py

```diff
@@ -1295,7 +1295,7 @@ def _faux_updates(_self: WLMLauncher, _names: t.List[str]) -> t.List[StepInfo]:
         ctx.setattr(SlurmLauncher, "get_step_update", get_faux_update(status_in))
 
         mani_handler = ManifestEventHandler("xyz", logger)
-        mani_handler.set_launcher("slurm", test_dir)
+        mani_handler.set_launcher("slurm")
```

Comment:
i'm sure i meant well when i introduced this ridiculous bug... ty for cleaning up my mess!